### PR TITLE
Issue #68, use engine-api instead of dockerclient for all multiTenant plugins

### PR DESCRIPTION
**Pull request by tsofnat**
use engine-api instead of dockerclient for  all multiTenant plugins

### CODE REVIEW

**Comment by ezrasilvera** [comment]
I don't think you need your own structure. I think many of the changes there can be avoided ...

**Comment by ezrasilvera** [comment]
Please have a look in   cluster\config.go  . Why don't you use the functions from there ? They already implemented what you need (e.g., ConsolidateResourceFields BuildContainerConfig, etc..)

Any special reason why you copied that code to uttils ?   Why not using cluster ?

**Comment by tsofnat** [comment]
I can use the cluster structure. 
I didn't want to depend on the cluster code if they will replace it.
if it is not an issue i will use the cluster.

File: pkg/multiTenancyPlugins/flavors/flavors.go [hunk 1]
```diff
@@ -84,11 +85,39 @@ func (flavorsImpl *DefaultFlavorsImpl) Handle(command utils.CommandEnum, cluster
 	if reqBody, _ := ioutil.ReadAll(r.Body); len(reqBody) > 0 {
 		var flavorIn Flavor
 		var buf bytes.Buffer
-		var containerConfig dockerclient.ContainerConfig
-		if err := json.NewDecoder(bytes.NewReader(reqBody)).Decode(&containerConfig); err != nil {
-			return err
```
Comment (ezrasilvera) [inline pkg/multiTenancyPlugins/flavors/flavors.go:89]: Why this was replaced with
var reqBody []byte
        defer r.Body.Close()
        if reqBody, _ = ioutil.ReadAll(r.Body); len(reqBody) > 0 {
            if err := json.NewDecoder(bytes.NewReader(reqBody)).Decode(&oldconfig); err != nil {
                log.Error(err)
                return err
            }
        }

**Pull request closed without merging**